### DISCUSSION TITLE
Triangle Line-Snap

### DISCUSSION THREAD

**Discussion by cgloeckner** [xournalpp/xournalpp#6004] [Q&A]
Hi,

is there a way to let the triangle snap to a line that was previously drawn with the triangle?

I'm using xournalpp in class as a teacher. Doing geometry works kind of well with it, but letting the triangle snap to a line would help to do things like enlarging the line afterwards or (if the radius-line of the compass would snap) placing the compass with more precision.

Is there such a feature?

Thanks in advance
glocke

**Comment by rolandlo** [comment]
Currently there is no such way. Enlarging a line afterwards can be done using the selection tool and stretching the line with one of the anchors at the corner.

**Reply by cgloeckner** [reply]
That's true but changes the line width accordingly (tested with a line that's not axis-aligned).

**Reply by rolandlo** [reply]
Use the option `Preserve line width` in the Drawing area tab of menu `Edit > Preferences`.
